src/lib/app-fns/tx/submitProposal.tsx

```diff
@@ -42,14 +43,11 @@ export const submitWhitelistProposalTx = ({
     }),
     ({ value: txInfo }) => {
       onTxSucceed?.();
-      const mimicLog: logs.Log = {
-        msg_index: 0,
-        log: "",
-        events: txInfo.events,
-      };
-      const txFee = findAttr(mimicLog, "tx", "fee");
+      const txFee = findAttribute(txInfo.events, "tx", "fee")?.value;
       const proposalId =
-        findAttr(mimicLog, "submit_proposal", "proposal_id") ?? "";
+        findAttribute(txInfo.events, "submit_proposal", "proposal_id")?.value ??
+        "";
```

Comment:
Again, the default case will never happen because `findAttribute` throws an error before.